src/components/Checkout/DonationItem.vue

```diff
@@ -55,7 +55,7 @@
 			</div>
 		</span>
 		<span class="small-3 show-for-small-only"></span>
-		<span class="small-9 medium-3 large-2 medium-text-font-size">
+		<span class="small-9 medium-4 large-4 medium-text-font-size">
```

Comment:
We could remove this large-4, but I'm not going to hold up this pull request for it. Will clean up afterward.